src/main/java/com/spotify/futures/FailFastWithThrowable.java

```diff
@@ -0,0 +1,59 @@
+package com.spotify.futures;
+
+import static java.util.Arrays.asList;
+
+import java.util.Collection;
+
+/**
+ * Fail fast implementation that completes exceptionally when one of the stages completes
+ * exceptionally matching a specific throwable class.
```

Comment:
Would it be possible to just always fail fast, without matching on a set of throwables here?

Reply:
Removed for good.